packages/progress-indicator/src/ProgressIndicator.tsx

```diff
@@ -57,7 +58,11 @@ export const ProgressIndicator = ({
 	const isMobile = (windowWidth || 0) <= tokens.breakpoint.lg - 1;
```

Comment:
```suggestion
	const isMobile = (windowWidth || 0) <= tokens.breakpoint.lg - 1;
	const subHeading = `${stepsCompleted} of ${totalSteps} steps completed`;
```

Reply:
I think you only need to generate this text once in this component, then it can be passed down to `ProgressIndicatorHeading ` and `ProgressIndicatorCollapseButton`.